consul/config.sls

```diff
@@ -13,3 +13,26 @@ write_{{ name }}_config:
     - watch_in:
       - service: start_consul_service
 {% endfor %}
+
+configure_systemd_resolved_conf:
+  file.append:
+    - name: /etc/systemd/resolved.conf
+    - text: |
+        DNS=127.0.0.1
+        Domains=~consul
+
+{% for protocol in ['udp', 'tcp'] %}
+redirect_{{ protocol }}_dns_port_to_consul_port:
+  iptables.append:
```

Comment:
Just curious if you tried out changing the port number in Consul to avoid needing the IPTables redirect.

Reply:
U mean having consul listen on 53?

Reply:
Yes, that way we don't even have to deal with iptables being in the loop.

Reply:
Removed iptables config and will add a separate PR for consul pillar to configure port.